cloud-sql-proxy/cloud-sql-proxy.sh

```diff
@@ -213,7 +217,15 @@ function main() {
   if [[ "${role}" == 'Master' ]]; then
     # Disable Hive Metastore and MySql Server.
     if (( ENABLE_CLOUD_SQL_METASTORE )); then
-      systemctl stop hive-metastore
+      local hive_metastore_status
+      hive_metastore_status=$(systemctl show --property=LoadState hive-metastore.service \
```

Comment:
Same comment here -- let's set a precedent for checking whether a service is enabled, rather than whether it's installed.